chart/compass/values.yaml

```diff
@@ -60,7 +60,7 @@ global:
         version: "PR-1156"
       provisioner:
         dir:
-        version: "PR-1193"
```

Comment:
This is an image of tests `global.images.tests.provisioner` :)

Reply:
That was dumb, indeed.
> "Mind that such missteps are the exception, and not the rule."